api/models/User.js

```diff
@@ -240,4 +252,96 @@ UserSchema.statics.getUserPictureAboutCategories = function (userId) {
   return this.findById(userId, 'picture about categories').exec();
 };
 
+// Get top ranked user ids for a given investor category, descending
+// Ranking done by total reps
+// Only returns the username, picture, rank, and reps for the category
+UserSchema.statics.findTopRankedInvestors = function(category) {
```

Comment:
These queries for findTop* are display info in the Leader and Trending tables. We don't need to slow ourselves down by taking all their categories and investments with us